api/config.js

```diff
@@ -78,6 +78,8 @@ overrides['DB_HOSTNAME'] = 'database.primary.host';
 overrides['DB_PORT'] = 'database.primary.port';
 overrides['REDIS_HOST'] = 'redis.host';
 overrides['REDIS_PORT'] = 'redis.port';
+overrides['RATELIMIT_LIMIT'] = 'ratelimiting.limit';
```

Comment:
Change the env vars to `RATELIMIT_COUNT` and `RATELIMIT_WINDOW` and adjust the config keys